src/devices/MatrixKeyboard/README.md

```diff
@@ -0,0 +1,11 @@
+# Matrix Keyboard
+
+An M×N matrix keyboard driver.
+
+## Summary
+
+You can connect any M×N Matrix Keyboard, theoretically, by using M+N GPIO pins, N pull-down resistors, and a ground pin.
```

Comment:
Can you list here at least a few devices that will work with this binding?

Reply:
Is this device something like [this](https://www.aliexpress.com/item/12-Key-Membrane-Switch-Keypad-4-x-3-Matrix-Array-Matrix-Keyboard-Module-Membrane-Switch-Keypad/32882310639.html?spm=2114.search0104.3.18.6b9a265f2tDG3M&ws_ab_test=searchweb0_0,searchweb201602_2_10065_10130_10068_10547_319_10546_317_10548_10545_10696_10084_453_454_10083_10618_10307_537_536_10059_10884_10887_321_322_10103,searchweb201603_52,ppcSwitch_0&algo_expid=87e31142-42d9-4d1b-a01a-f1fa7f637365-2&algo_pvid=87e31142-42d9-4d1b-a01a-f1fa7f637365&transAbTest=ae803_5)

if so please add a picture (make sure you got permissions for the pic or the license allows it)

Reply:
I'm not sure if I can use current picture safely. If not, I can make my own keyboard and get some.
(yes, it's right on my schedule

Reply:
> Is this device something like this
> if so please add a picture (make sure you got permissions for the pic or the license allows it)

I'm not sure if I can use current picture safely. If not, I can make my own keyboard and get some.
(yes, it's right on my schedule

Reply:
I'm not sure if I can use current picture safely. If not, I can make my own keyboard and take some picture.
(yes, it's right on my schedule